src/main/resources/templates/progress/pass.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="cz">
+<head>
+    <meta charset="UTF-8">
+    <title>Stage ID passed</title>
```

Comment:
Klidně si myslím, že můžem držet všude češtinu.

Reply:
@mjanousek Teda myslím, že to Jíťa kvůli tomu předělávala, ne?

Reply:
upraveno do češtiny v následujícím commitu